### PR TITLE
Updated broken example recipe links.

### PR DESCRIPTION
**Pull request by tttc3**
### Description
Updates some broken links to example conda recipes. Also adds a link to the conda-forge feedstocks and AnacondaRecipes aggregated feedstocks in addition to the link to the now archived conda-recipes repo.

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [x] ~~Add / update necessary tests?~~
- [x] Add / update outdated documentation?

### CODE REVIEW

File: docs/source/concepts/recipe.rst [hunk 1]
```diff
@@ -76,9 +76,10 @@ Conda-build performs the following steps:
 
    #. Runs the test scripts.
 
-The `conda-recipes
-<https://github.com/continuumio/conda-recipes>`_ repo
-contains example recipes for many conda packages.
+The `conda-recipes <https://github.com/continuumio/conda-recipes>`_ repo,
```
Comment (kathatherine) [inline docs/source/concepts/recipe.rst:79]:
```suggestion
The archived `conda-recipes <https://github.com/continuumio/conda-recipes>`_ repo,
```

Comment (kathatherine) [reply docs/source/concepts/recipe.rst:79]: Should probably be clear it's an archive.

**Review by kathatherine — CHANGES_REQUESTED** [review]
Thank you so much for this! Just one suggestion.

**Review by kathatherine — DISMISSED** [review]
Thanks! Looks good to me.

File: news/4580-update-broken-links [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+* <news item>
+
+### Deprecations
+
+* <news item>
+
+### Docs
+
+* Updated broken links to example conda recipes and updated link to the now archived conda-recipes, with additional links to AnacondaRecipes aggregated feedstocks and conda-forge feedstocks.
```
Comment (kenodegard) [inline news/4580-update-broken-links:15]:
```suggestion
* Updated broken links to example conda recipes and updated link to the now archived conda-recipes, with additional links to AnacondaRecipes aggregated feedstocks and conda-forge feedstocks. (#4580)

```

**Review by kenodegard — APPROVED** [review]

**Pull request merged**